webots_ros2_epuck/launch/robot_with_tools_launch.py

```diff
@@ -27,16 +27,16 @@
 def generate_launch_description():
     package_dir = get_package_share_directory('webots_ros2_epuck')
 
-    use_nav = LaunchConfiguration('nav', default=False)
-    use_rviz = LaunchConfiguration('rviz', default=False)
-    use_mapper = LaunchConfiguration('mapper', default=False)
-    synchronization = LaunchConfiguration('synchronization', default=False)
+    use_nav = LaunchConfiguration('nav', default=True)
+    use_rviz = LaunchConfiguration('rviz', default=True)
+    use_mapper = LaunchConfiguration('mapper', default=True)
+    synchronization = LaunchConfiguration('synchronization', default=True)
```

Comment:
Why are these changed?

Reply:
It was to for testing, forget to revert.